internal/characters/yaemiko/kitsune.go

```diff
@@ -142,17 +143,32 @@ func (c *char) kitsuneTick(totem *kitsune) func() {
 		if c.Base.Cons >= 6 {
 			ai.IgnoreDefPercent = 0.60
 		}
-		enemy := c.Core.Combat.RandomEnemyWithinArea(totem.kitsuneArea, nil)
-		if enemy != nil {
+
+		// spawn 1 attack
+		// priority: enemy > gadget
+		tick := func(pos geometry.Point) {
 			c.Core.QueueAttack(
 				ai,
-				combat.NewCircleHitOnTarget(enemy, nil, 0.5),
+				combat.NewCircleHitOnTarget(pos, nil, 0.5),
 				1,
 				1,
 				c.particleCB,
 				c4cb,
 			)
 		}
+
+		var pos geometry.Point
+		// try to target an enemy first
+		enemy := c.Core.Combat.RandomEnemyWithinArea(totem.kitsuneArea, nil)
+		if enemy != nil {
+			pos = enemy.Pos()
+		} else {
+			// target gadget if no enemy was targeted
+			gadget := c.Core.Combat.RandomGadgetWithinArea(totem.kitsuneArea, nil)
+			pos = gadget.Pos()
+		}
+		tick(pos)
```

Comment:
Need a `nil` check for case of no gadget is found and `pos` remained nil/unspecified